packages/typespec-azure-vscode/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "typespec-azure-vscode",
+  "displayName": "typespec azure",
```

Comment:
@josefree , it's better to have a better name to make sure 3rd party user won't be confused.